src/assets/blog/20240906-ticket-registration.md

```diff
@@ -0,0 +1,57 @@
+---
+template: 'blog'
+slug: 'ticket-registration'
+title: 'Ticket Registration Process'
+author: 'Michael Richardson'
+publishedDate: '2024-09-06'
+published: 'true'
+summary: 'Have questions about buying and registering tickets for Momentum? This should help make the process easy and stress-free.'
+---
+
+# Momentum Registration Guide
+
+You want tickets for Momentum 2024. That's great! We want to make it easy and convenient for you to get tickets for Momentum 2024. This guide will hopefully answer some of your questions. If you're ever unsure, just drop an email to tickets@momentumdevcon.com and we'll get you sorted right away.
+
+# Step 1: Buy the Tickets
+Tickets are conveniently available through our website, https://momentumdevcon.com.
+We use Tito as our ticketing provider to facilitate a smooth and simple process.
+
+You can purchase tickets via PayPal (using credit card or any other payment they offer) or via invoice. PayPal is processed instantly. Invoice requires you to mail us a check. 
+
+We recommend PayPal. It's a lot less hassle and we don't have to track you down to make sure you send the payment!
+
+# Step 2: Allocate the tickets
+## I'm buying a Momentum ticket for myself
+
+This is simple! Buy your ticket, assign it to yourself, answer a few questions, no big deal. You can answer the questions later instead **but** *we will nag you about it if you forget*.
+
+## I'm buying Momentum tickets for others
+
+Don't panic, with a simple trick, this process is easier than you expect! All you need are the attendees' names and emails. They will fill out the rest.
+
+Once you purchase the tickets, you'll see this prompt:
```

Comment:
Need a newline between text and image else the image would be in-line
![image](https://github.com/user-attachments/assets/6a570a49-d029-457a-a4f1-58394809cd8e)

Reply:
Thanks for calling that out